ci/jenkins/test-mc.sh

```diff
@@ -442,14 +442,12 @@ function run_multicluster_e2e {
     fi
 
     set -x
-    go test -v antrea.io/antrea/multicluster/test/e2e --logs-export-dir `pwd`/antrea-multicluster-test-logs $options
+    go test -v antrea.io/antrea/multicluster/test/e2e $options
     if [[ "$?" != "0" ]]; then
         TEST_FAILURE=true
     fi
     set +x
     set -e
-
-    tar -zcf antrea-test-logs.tar.gz antrea-multicluster-test-logs
```

Comment:
There is actually not used at all in multi-cluster e2e since there is no `exportLogs` method defined in MC e2e codes. The file will always be empty. I will evaluate if there is any necessary to define and export logs for MC e2e, but I think we can remove it safely here since there is no logs generated by MC e2e.